debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -67,6 +67,71 @@ public static DeleteHandling parse(String value, String defaultValue) {
         }
     }
 
+    public enum DeleteHandlingV2 implements EnumeratedValue {
+        DROP("drop"),
+        TOMBSTONE("tombstone"),
+        REWRITE("rewrite"),
+        REWRITE_WITH_TOMBSTONE("rewrite-with-tombstone");
+
+        private final String value;
+
+        DeleteHandlingV2(String value) {
+            this.value = value;
+        }
+
+        @Override
+        public String getValue() {
+            return value;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value the configuration property value; may not be null
+         * @return the matching option, or null if no match is found
+         */
+        public static DeleteHandlingV2 parse(String value) {
+            if (value == null) {
+                return null;
+            }
+            value = value.trim();
+            for (DeleteHandlingV2 option : DeleteHandlingV2.values()) {
+                if (option.getValue().equalsIgnoreCase(value)) {
+                    return option;
+                }
+            }
+            return null;
+        }
+
+        /**
+         * Determine if the supplied value is one of the predefined options.
+         *
+         * @param value the configuration property value; may not be null
+         * @param defaultValue the default value; may be null
+         * @return the matching option, or null if no match is found and the non-null default is invalid
+         */
+        public static DeleteHandlingV2 parse(String value, String defaultValue) {
+            DeleteHandlingV2 mode = parse(value);
+            if (mode == null && defaultValue != null) {
+                mode = parse(defaultValue);
+            }
+            return mode;
+        }
+    }
+
+    public static final Field HANDLE_DELETES_V2 = Field.create("delete.handling.mode.v2")
+            .withDisplayName("Handle delete records")
+            .withEnum(DeleteHandlingV2.class)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .optional()
+            .withDescription("How to handle delete records. Options are: "
+                    + "drop - will remove both delete and tombstone,"
+                    + "tombstone - will convert delete to tombstone and remove tombstone (should be the default),"
```

Comment:
What's the difference between the first tombstone and the second tombstone word here?
I assume they don't mean the same event,right?
I think it is a technical description about what happens under the hood, is it?

I assume that option means that Debezium drops the delete event and only emits a tombstone? Why not explaining it like this, for the user it should not matter if technically a delete event is converted into a tombstone and then gets emitted and the original tombstone is dropped (in order to only emit a single tombstone instead of 2).